install/install.go

```diff
@@ -1063,6 +1434,62 @@ func K8sUninstaller(c *k8s.Client, o Options) error {
 	return nil
 }
 
+// K8sUninstaller for karmor uninstall
+func K8sUninstaller(c *k8s.Client, o Options) error {
+	var ns string
+	var kocName string = "kubearmorconfigs.operator.kubearmor.com"
+	settings := cli.New()
+
+	actionConfig := actionConfigInit("", settings)
+	statusClient := action.NewStatus(actionConfig)
+	res, err := statusClient.Run("kubearmor-operator")
+	if err != nil {
+		fmt.Println("ℹ️  Helm release not found. Switching to legacy uninstaller.")
+		return err
+	}
+	ns = res.Namespace
+
+	actionConfig = actionConfigInit(ns, settings)
+	client := action.NewUninstall(actionConfig)
+	client.Timeout = 5 * time.Minute
+	client.DeletionPropagation = "background"
+
+	log.SetOutput(io.Discard)
+	_, err = client.Run("kubearmor-operator")
+	log.SetOutput(os.Stdout)
+	if err != nil {
+		fmt.Println("ℹ️  Error uninstalling through Helm. Switching to legacy uninstaller.")
+		return err
+	}
+
+	if o.Force {
+		operatorClientSet, err := operatorClient.NewForConfig(c.Config)
+		if err != nil {
+			return fmt.Errorf("failed to create operator clientset: %w", err)
+		}
+
+		fmt.Printf("❌\tRemoving CR kubearmorconfig-default\n")
+		if err := operatorClientSet.OperatorV1().KubeArmorConfigs(ns).Delete(context.Background(), "kubearmorconfig-default", metav1.DeleteOptions{}); apierrors.IsNotFound(err) {
+			fmt.Printf("CR %s not found\n", kocName)
+		}
+
+		fmt.Printf("❌\tRemoving CRD %s\n", kocName)
+		if err := c.APIextClientset.ApiextensionsV1().CustomResourceDefinitions().Delete(context.Background(), kocName, metav1.DeleteOptions{}); apierrors.IsNotFound(err) {
+			fmt.Printf("CRD %s not found\n", kocName)
+		}
+
+		removeAnnotations(c, ns)
+	}
+
+	fmt.Println("❌\tKubeArmor resources removed")
```

Comment:
Can we add a statement here, 
```
If !o.Force
Resources not managed by helm/GLobal Resources are not cleaned up. Please use karmor uninstall --force if you want complete cleanup
```

WDYT?

Reply:
Sounds good. I will take a look.